arch-update/arch-update

```diff
@@ -37,10 +39,7 @@ def create_argparse():
     parser.add_argument(
         '-a',
         '--aur',
-        action = 'store_const',
-        const = True,
-        default = _default('AUR', 'False', strbool),
-        help='Include AUR packages. Attn: Yaourt must be installed'
```

Comment:
The configuration should be done using environment variables (see #132). The arguments are just kept for backwards compatibility. Also, please ensure that the functionality stays backwards compatible: `AUR=true` should still work.